Gemfile

```diff
@@ -4,14 +4,15 @@ source 'https://rubygems.org'
 
 ruby File.read('.ruby-version')
 
-gem 'rails', '6.1.3.2'
+gem 'rails', '6.1.4.1'
 
 gem 'pg'
-gem 'puma'
+gem 'puma', '~> 5.4.0'
```

Comment:
I'm curious about this pin - why not leave it off?

Reply:
It's not necessary, I just have a habit of being pretty specific about which versions of dependencies are in use. I went ahead and removed this line since the rest of the dependencies aren't quite so specific.